src/Promitor.Integrations.Sinks.OpenTelemetry/Configuration/OpenTelemetryCollectorSinkConfiguration.cs

```diff
@@ -1,7 +1,16 @@
-﻿namespace Promitor.Integrations.Sinks.OpenTelemetry.Configuration
+﻿using OpenTelemetry.Exporter;
+
+namespace Promitor.Integrations.Sinks.OpenTelemetry.Configuration
 {
     public class OpenTelemetryCollectorSinkConfiguration
     {
         public string CollectorUri { get; set; }
+        
+        public CollectorInfo Collector { get; set; }
+
+        public class CollectorInfo
+        {
+            public string CollectorProtocol { get; set; } = "grpc";
```

Comment:
Let's use the official enum instead; less responsibility for us
```suggestion
            public OtlpExportProtocol CollectorProtocol { get; set; } = default(OtlpExportProtocol);
```

Reply:
swapped to enum with default